### ISSUE TITLE
Can't create new tickets

### ISSUE THREAD

**Issue by Botanium** [frappe/desk#752]
I just did a fresh install for V14, and I get below error

### App Versions
```
{
	"chat": "0.0.1",
	"erpnext": "14.3.0",
	"erpnext_telegram_integration": "1.0.1",
	"frappe": "14.11.1",
	"frappedesk": "0.4.6",
	"payments": "0.0.1",
	"twilio_integration": "0.0.1"
}
```
### Route
```
Form/Ticket/new-ticket-2
```
### Trackeback
```
Traceback (most recent call last):
  File "apps/frappe/frappe/app.py", line 69, in application
    response = frappe.api.handle()
  File "apps/frappe/frappe/api.py", line 54, in handle
    return frappe.handler.handle()
  File "apps/frappe/frappe/handler.py", line 45, in handle
    data = execute_cmd(cmd)
  File "apps/frappe/frappe/handler.py", line 83, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "apps/frappe/frappe/__init__.py", line 1585, in call
    return fn(*args, **newargs)
  File "apps/frappe/frappe/client.py", line 225, in save
    doc.save()
  File "apps/frappe/frappe/model/document.py", line 303, in save
    return self._save(*args, **kwargs)
  File "apps/frappe/frappe/model/document.py", line 325, in _save
    return self.insert()
  File "apps/frappe/frappe/model/document.py", line 283, in insert
    self.run_post_save_methods()
  File "apps/frappe/frappe/model/document.py", line 1089, in run_post_save_methods
    self.run_method("on_update")
  File "apps/frappe/frappe/model/document.py", line 930, in run_method
    out = Document.hook(fn)(self, *args, **kwargs)
  File "apps/frappe/frappe/model/document.py", line 1270, in composer
    return composed(self, method, *args, **kwargs)
  File "apps/frappe/frappe/model/document.py", line 1254, in runner
    add_to_return_value(self, f(self, method, *args, **kwargs))
  File "apps/frappe/frappe/automation/doctype/assignment_rule/assignment_rule.py", line 263, in apply
    new_apply = assignment_rule.apply_assign(doc)
  File "apps/frappe/frappe/automation/doctype/assignment_rule/assignment_rule.py", line 51, in apply_assign
    return self.do_assignment(doc)
  File "apps/frappe/frappe/automation/doctype/assignment_rule/assignment_rule.py", line 57, in do_assignment
    user = self.get_user(doc)
  File "apps/frappe/frappe/automation/doctype/assignment_rule/assignment_rule.py", line 93, in get_user
    return self.get_user_round_robin()
  File "apps/frappe/frappe/automation/doctype/assignment_rule/assignment_rule.py", line 106, in get_user_round_robin
    return self.users[0].user
IndexError: list index out of range

```
### Request Data
```
{
	"type": "POST",
	"args": {
		"doc": "{\"docstatus\":0,\"doctype\":\"Ticket\",\"name\":\"new-ticket-1\",\"__islocal\":1,\"__unsaved\":1,\"owner\":\"botan.b.abdullah@gmail.com\",\"status\":\"Open\",\"priority\":\"Medium\",\"agreement_status\":\"First Response Due\",\"opening_date\":\"2022-10-20\",\"via_customer_portal\":0,\"feedback_submitted\":0,\"__run_link_triggers\":1,\"subject\":\"Testing\",\"raised_by\":\"botan.b.abdullah@gmail.com\",\"description\":\"<div class=\\\"ql-editor read-mode\\\"><p>Tesing purpose only</p></div>\"}"
	},
	"freeze": true,
	"headers": {},
	"error_handlers": {},
	"url": "/api/method/frappe.client.save"
}
```
### Response Data
```
{
	"exception": "IndexError: list index out of range"
}
```

**Comment by kittiu** [comment]
I face, the same issue. Then, I add an Agent and it works.

**Comment by Botanium** [comment]
Interesting, let me try again and see if I can reproduce it. Can you tell me what versions of apps are you using on your bench?

**Comment by Botanium** [comment]
![image](https://user-images.githubusercontent.com/3994826/197463428-73c1eef4-2d06-49f2-ba0d-f0ee16e573f9.png)


I get the same error without agent creation, but when I create the agent, Everything works fine.

**Comment by kittiu** [comment]
I use frappecloud. But I still can't get /frappedesk to work only basic ticket (I guess there are 2 interface old and new). I go this, but even I add default email account, still not work.

![image](https://user-images.githubusercontent.com/1973598/197472781-186e187b-1712-4d11-92cf-66ce50c56c12.png)

Do you experience the same?

**Comment by Botanium** [comment]
Not yet, but I will dig more and see what other issues might occur. I think you should make another issue if you encounter them. Hopefully the team will pick it up.

**Comment by joshuarestivo** [comment]
Creating an Agent cleared this error for me.

**Comment by harshit-30** [comment]
I am closing this assuming it has been resolved in our current releases. If it's not, please reopen with current tracebacks and steps to reproduce.
